cypress/integration/brewery/Router_Redirection.spec.js

```diff
@@ -0,0 +1,38 @@
+import { PAGE_NAME } from '../../commons/constants/generic/TestConstants';
+import { Login, ScenarioManager, Scenarios } from '../../commons/actions';
+import { setup } from '../../commons/utils/setup';
+import { stub } from '../../commons/services/stubbing';
+import { DEFAULT_SCENARIOS_LIST } from '../../fixtures/stubbing/default';
+import { apiUtils as api } from '../../commons/utils';
+
+describe('Redirects to right page', () => {
+  before(() => {
+    setup.initCypressAndStubbing();
+    stub.start({
+      GET_DATASETS: true,
+      GET_SCENARIOS: true,
+      GET_WORKSPACES: true,
+      GET_SOLUTIONS: true,
+    });
+  });
+
+  after(() => {
+    stub.stop();
+  });
+
+  it('redirects to scenario manager view after login', () => {
+    Login.login(`W-stbbdbrwry${PAGE_NAME.SCENARIO_MANAGER}`);
+    ScenarioManager.getScenarioManagerView().should('be.visible');
+  });
+
+  it('redirects from scenario manager to scenario view with chosen scenario as current', () => {
+    Login.login();
+    Scenarios.getScenarioView().should('be.visible');
+    ScenarioManager.switchToScenarioManager();
+    ScenarioManager.getScenarioViewRedirect(DEFAULT_SCENARIOS_LIST[3].id).should('exist');
+    api.interceptGetScenario(DEFAULT_SCENARIOS_LIST[3].id);
+    ScenarioManager.getScenarioViewRedirect(DEFAULT_SCENARIOS_LIST[3].id).click();
+    cy.url({ timeout: 5000 }).should('include', `/scenario/${DEFAULT_SCENARIOS_LIST[3].id}`);
```

Comment:
Maybe we could define a generic action `openScenario(scenarioId)` (or `visitScenario` or `redirectToScenario`) in the ScenarioManager actions to factorize these 3 lines? We will probably reuse the redirection feature in other tests.